src/backend/ceph.rs

```diff
@@ -484,19 +455,13 @@ impl CephBackend {
         ceph_bluestore_tool(&lv_dev_name, &mount_point, simulate)?;
 
         let host_info = Host::new()?;
-        let gb_capacity = vg_size / 1_073_741_824;
```

Comment:
What is this magic number?

Reply:
Hah I have no clue.

Reply:
That was probably something I wrote.  I'd have to guess it's 1GB in bytes

Reply:
yups.  its 1 Gibibyte exactly